zilencer/migrations/0021_remotezulipserver_create_audit_log_backfill.py

```diff
@@ -0,0 +1,49 @@
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.utils.timezone import now as timezone_now
+
+
+def backfill_remote_zulip_server_creation_log_events(
+    apps: StateApps, schema_editor: DatabaseSchemaEditor
+) -> None:
+    RealmAuditLog = apps.get_model("zerver", "RealmAuditLog")
+    RealmAuditLog.REMOTE_SERVER_CREATED = 10215
+
+    RemoteZulipServer = apps.get_model("zilencer", "RemoteZulipServer")
+    RemoteZulipServerAuditLog = apps.get_model("zilencer", "RemoteZulipServerAuditLog")
+
+    objects_to_create = []
+    for remote_server in RemoteZulipServer.objects.all():
+        entry = RemoteZulipServerAuditLog(
+            server=remote_server,
+            event_type=RealmAuditLog.REMOTE_SERVER_CREATED,
+            event_time=timezone_now(),
```

Comment:
I noticed `RemoteZulipServer` has the field `last_updated` - I think we could use that for `event_time`? That should actually match the real creation time, because i don't think we actually do anything to the `last_updated` value aside of it being set automatically upon creation